app/config.py

```diff
@@ -56,6 +70,10 @@ class Test(Config):
 
     FRONTEND_HOSTNAME = "document-download-frontend-test"
 
+    REDIS_URL = os.getenv("REDIS_URL", "redis://localhost:6379/0")
```

Comment:
Just wanted to check on a slight difference between this and the admin and if it is intentional

For admin and the `Test` class we have:

```
REDIS_URL = 'redis://localhost:6379/0'
REDIS_ENABLED = os.environ.get('REDIS_ENABLED') == '1'
```

which is inherited from the `Development` class: https://github.com/alphagov/notifications-admin/blob/master/app/config.py#L118-L119

Whereas in this app for the `Test` class we have:

```
REDIS_URL = os.getenv("REDIS_URL", "redis://localhost:6379/0")
REDIS_ENABLED = REDIS_URL is not None
```

Am I right in thinking this means that for admin, redis is not enabled for the `Test` class by default whereas for doc download API we would have redis enabled by default? If so, is there a reason for us to diverge from how admin does?

Reply:
Not sure if this is outdated or not, but for this app we just always have REDIS_ENABLED=1. It's possible your snippet `REDIS_ENABLED = REDIS_URL is not None` was from an old copy of my branch you had locally.

Reply:
Maybe, as you say, it's nicer to have redis toggled on explicitly for tests+local dev, so will add the `REDIS_ENABLED = os.environ.get('REDIS_ENABLED') == '1'` line to Test+Development configs.